easybuild/easyconfigs/s/shapeit/shapeit-2.r837.GLIBCv2.12_linux_x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'shapeit'
+version = '2.r837.GLIBCv2.12'
```

Comment:
'.GLIBCv2.12' part was attached to source that the Easyconfig pulls(and it changes for different versions, like currently on their website there is shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz and shapeit.v2.r837.GLIBCv2.20.Linux.static.tgz), and I wanted to make it more agile so that when a new version comes in for SHAPEIT, we would just need to change the variable values at the top, rather then changing any part of the Easyconfig below it(though, that doesn't take much time). But, you are right in saying that it would shorten the modulefile's name. 

Also, SHAPEIT (uppercase) is fine. There was no particular reason I chose it in lowercase. :)  I'd change that .

So, let me know what you think about GLIBC part, and I can make the changes